ledger-core/instrumentation/insapp/init_components.go

```diff
@@ -105,7 +107,7 @@ func (s *Server) initComponents(ctx context.Context, cfg configuration.Configura
 		ns = nsn
 
 		pulses = memstor.NewStorageMem()
-		pm := NewPulseManager()
+		pm := component2.NewPulseManager()
```

Comment:
component2 -> app_component, for example

Reply:
done